src/OrchardCore.Modules/OrchardCore.AuditTrail/OrchardCore.AuditTrail.csproj

```diff
@@ -17,6 +17,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Module.Targets\OrchardCore.Module.Targets.csproj" />
```

Comment:
Add these analyzers temporarily (remove them before merge) to pinpoint important issues:

```xml
  <ItemGroup>
    <PackageReference Include="AsyncFixer" Version="1.5.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.9.45-alpha">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="SecurityCodeScan" Version="3.5.4">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>
```

There are a couple of interesting ones but simple to fix:
![image](https://user-images.githubusercontent.com/1976647/108240060-c580c000-714a-11eb-8d92-e581f189111d.png)